velox/functions/prestosql/Arithmetic.h

```diff
@@ -229,7 +229,7 @@ struct PowerFunction {
   template <typename TInput>
   FOLLY_ALWAYS_INLINE void
   call(double& result, const TInput& a, const TInput& b) {
-    result = (std::isnan(a) && b != 0) || std::isnan(b) || std::isinf(b)
+    result = (std::isnan(a) && b != 0) || std::isnan(b)
```

Comment:
why do we need this condition? `(std::isnan(a) && b != 0)`
does std::pow() return wrong result under this condition?

Reply:
This actually isn't needed. pow() returns the correct result in that case. I will remove the condition.